### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Multi combobox] The display of the listbox is not perceptible with the screen reader.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10828]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The display of the listbox is not perceptible with the screen reader.
Screenreader should announce the list that gets opened.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular 15
core platform 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/multi-combobox#data-source

**Comment by nikolay-kolarov** [comment]
Hello @I543348 ,

I am not sure if I get your point.
I checked the implementation and there are three things that happen:
- When you start typing and the list is opened, you receive an "expanded" state announcement, which indicates that the attached popup to the combobox is opened/expanded.
- When you start typing and the list is opened, you receive information about how many list items are found based on your input.
- When you go with the arrows in the list, then you receive the information that this is a listbox, etc...

So could you please specify what exactly is missing and in which interaction with the control?

I also cross checked with UI5 and there the behavior is similar: https://ui5.sap.com/#/entity/sap.m.MultiComboBox/sample/sap.m.sample.MultiComboBox

@droshev , @InnaAtanasova - FYI

Thanks,
Nikolay

**Comment by sagar7259** [comment]
Hi @nikolay-kolarov,

The information about the search results list is being read to the users, the issue is not that.

Example: i will be searching the character 'S' which gives 2 results "Strawberry" and "Spinach"

Speech history:
"expanded
selected
pinach edit combo
2 result list items. Press escape to exit the input field and use the left and right arrow keys to navigate through the selected options"

Observed behaviour:
The spinach is not at all selected, but its reading has selected.
and if you can see the tutor message about the user of arrow keys, its been reading the information only about the tokens but not about the flat list displayed. Here to navigate within the flat list up/down arrows are used and user is not conveyed anything about how to access and navigate within the flat list.

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hi @sagar7259 ,

Thanks for the details. Now I understand what you mean.
The selected state is not exposing the fact that the item is selected, but is announced because the auto-completed/highlighted text is selected and it reads even only the selected part. Please check the green text in the speech output below. Also when pressing Tab if the value matches a token it has to be tokenized (but as far as I remember this will be handled in another issue @droshev  - could you please double check?). So there is no expected change for this.

Now, let`s focus on the list. Recently the colleagues have added the text for the navigation in the token, but you are right that the info for arrow up and down is missing.
Here is the same extract from UI5 where in red you can see the two texts (the first one is shorter as there are still not tokens and the second is the full version):
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/f716fd99-d02b-4cb4-9d13-941788117a48)

So, @droshev  and team, could you please use this as a reference and add/change this in the implementation? At the moment when there are no selected tokens the user receive the guidance to use lefr/right arrow keys, which is wrong (this message needs to be announced only when there are tokens). And the part to use up/down keys to navigate in the suggestions is missing.

Thanks,
Nikolay

Thanks,
Nikolay
